parallelMatrixMultiplication/parallelMatrixMultiplication/CreateTable.cs

```diff
@@ -164,13 +215,26 @@ private static void MultiplyingMatricesWithNegativeNumbers(string filePath)
 
     private static void MultiplyingMatricesWithZero(string filePath)
     {
-        var sizeThreads = 10;
-        var listOfValuesFirstMatrix = new List<int> { 1, 2, 0, 4, 5, 6, 0, 8, 9, 10 };
-        var listOfValuesSecondMatrix = new List<int> { 1, 2, 0, 0, 5 };
-        var listOfCorrectValues = new List<int> { 30, 56 };
-        var firstMatrix = Matrix.CreateMatrix(2, 5, listOfValuesFirstMatrix);
-        var secondMatrix = Matrix.CreateMatrix(5, 1, listOfValuesSecondMatrix);
-        var correctMatrix = Matrix.CreateMatrix(2, 1, listOfCorrectValues);
+        var sizeThreads = Environment.ProcessorCount;
+        var listOfValuesFirstMatrix = new List<int[]> { };
+        var listOfValuesSecondMatrix = new List<int[]> { };
+        var listOfCorrectValues = new List<int[]> { };
+
+        listOfValuesFirstMatrix.Add(new int[5] { 1, 2, 0, 4, 5 });
+        listOfValuesFirstMatrix.Add(new int[5] { 6, 0, 8, 9, 10 });
+
+        listOfValuesSecondMatrix.Add(new int[1] { 1 });
+        listOfValuesSecondMatrix.Add(new int[1] { 2 });
+        listOfValuesSecondMatrix.Add(new int[1] { 0 });
+        listOfValuesSecondMatrix.Add(new int[1] { 0 });
+        listOfValuesSecondMatrix.Add(new int[1] { 5 });
```

Comment:
Можно проще (здесь и в других местах)

```suggestion
var listOfValuesSecondMatrix = new List<int[]>
{
    new int[1] { 1 },
    new int[1] { 2 },
    new int[1] { 0 },
    new int[1] { 0 },
    new int[1] { 5 },    
}
```